README.md

```diff
@@ -51,31 +43,10 @@ Blaze4J bridges Java and native C++ for **fast, standards-compliant JSON Schema
 - **Easy integration**: Simple API for compiling schemas and validating instances.
 - **Cross-platform**: Works on Windows, Linux, and macOS.
 
----
-
-## Supported Drafts 
-
-| Draft Version | Status        | Specification                                                                 |
-|---------------|--------------|-------------------------------------------------------------------------------|
-| Draft 2020-12 | ✅ Supported | [Spec](https://json-schema.org/draft/2020-12/release-notes.html)              |
-| Draft 2019-09 | ✅ Supported | [Spec](https://json-schema.org/draft/2019-09/release-notes.html)              |
-| Draft 7       | ✅ Supported | [Spec](https://json-schema.org/draft-07/release-notes.html)                   |
-| Draft 6       | ✅ Supported | [Spec](https://json-schema.org/draft-06/release-notes.html)                   |
-| Draft 4       | ✅ Supported | [Spec](https://json-schema.org/draft-04/release-notes.html)                   |
-
-
----
-<a name="supported-resolvers"></a>
-## Supported `$ref` Resolvers:
-
-| Resolver Type | Status        | Description                                   |
-|---------------|---------------|-----------------------------------------------|
-| `http`        | ✅ Supported  | Resolves external schemas via HTTP(S) URLs    |
-| `classpath`   | ✅ Supported  | Resolves schemas from the Java classpath      |
+--- 
+## Project Structure
```

Comment:
This also doensn't look necessary for users. Also, anyone can get the directory structure, if the need it, with a `tree` command, so there is no need to include it in the docs. Please remove it (both from here and the TOC)